index.html

```diff
@@ -208,10 +234,11 @@ <h1 class="skills-heading">Skills</h1>
         <section class="contact-form">
             <img alt="Logo" class="geometry" src="./images/image_geometry_1.png">
             <h2 class="contact-heading">Get started</h2>
-            <p class="contact-paragraph">Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit,
-                sed do eiusmod tempor incididunt
-                ut labore et dolore magna.</p>
+            <p class="contact-paragraph">Lorem ipsum dolor sit amet, 
+                consectetur adipiscing elit, sed do eiusmod tempor incididunt 
+                ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+                quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
+                ea commodo consequat. </p>
             <form class="form" action="https://formspree.io/f/mjvlwyao" method="POST">
```

Comment:
Please try to match the Figma design in a mobile version for the email input
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/154376433-36851eca-a92b-4c18-bef9-565b25e5323e.png) | ![image](https://user-images.githubusercontent.com/24703171/154376482-77193438-fd4f-4c09-a50f-c35e5d620119.png) |